projects/unisave/index.js

```diff
@@ -0,0 +1,30 @@
+const { getChainTvl } = require('../helper/getUniSubgraphTvl');
+const { staking } = require("../helper/staking");
+const { pool2 } = require("../helper/pool2");
+const {uniTvlExport} = require("../helper/calculateUniTvl");
+
+const Factory = "0x32CE36F6eA8d97f9fC19Aab83b9c6D2F52D74470";
+const stakingContract = "0xe5b677c7aae044135f81FEb4c38b877272e6F6dd";
+const Y3D = "0xc7fD9aE2cf8542D71186877e21107E1F3A0b55ef";
+const WETH_Y3D_LP = "0xd0f52b7e4ef306b1a0b7d5a3206dfd7f7e032e2e";
+
+const stakingContractBSC = "0x3e8a3E3A7132311f66B36e6A91A6928C273f0077";
+const Y3D_BSC = "0x12e2fcfa079fc23ae82ab82707b402410321103f";
+
+const chainTvl = getChainTvl({
+    ethereum: 'https://api.thegraph.com/subgraphs/name/kodamasakuno/uniscam-swap',
+})
+
+module.exports = {
+    methodology: "TVL accounts for the liquidity on all AMM pools.",
+    misrepresentedTokens: true,
+    ethereum: {
+        staking: staking(stakingContract, Y3D),
+        pool2: pool2(stakingContract, WETH_Y3D_LP),
+        tvl: chainTvl('ethereum')
+    },
+    bsc: {
+        staking: staking(stakingContractBSC, Y3D_BSC,"bsc"),
+        tvl: uniTvlExport(Factory, "bsc"), // Decimals or data are incorrects on the getReserves of Pair Contracts!!
```

Comment:
is the tvl currently broken?

Reply:
They show correctly the tvl on them webside I think. But when I went to use the pairs on the Factoory address to get data from getReserves I checked that the value was incorrect comparing with what it must to be, and it's bigger and different, because, I think the problem on getReserves is not the decimals, I think they are adding sommething else that are not the balances of each token holding by the pair or they're adding them incorrectly.